vhdbuilder/packer/install-dependencies.sh

```diff
@@ -189,23 +185,14 @@ if [[ $OS == $UBUNTU_OS_NAME && $(isARM64) != 1 ]]; then  # no ARM64 SKU with GP
       fi
     fi
   else
-    if grep -q "fullgpu" <<< "$FEATURE_FLAGS"; then
-      bash -c "$DOCKER_GPU_INSTALL_CMD $NVIDIA_DRIVER_IMAGE:$NVIDIA_DRIVER_IMAGE_TAG $gpu_action"
-      ret=$?
-      if [[ "$ret" != "0" ]]; then
-        echo "Failed to install GPU driver, exiting..."
-        exit $ret
-      fi
-    fi
+    echo "Unsupported container runtime"
+    exit 1
   fi
 fi
 
 if [ "${CONTAINER_RUNTIME:=}" == "containerd" ]; then
     systemctlEnableAndStart containerd-monitor.timer || exit $ERR_SYSTEMCTL_START_FAIL
-else
-    systemctlEnableAndStart docker-monitor.timer || exit $ERR_SYSTEMCTL_START_FAIL
 fi
-
```

Comment:
maybe also fail hard here if container runtime isn't containerd?

Reply:
following from what Cam said, we seem to be using
`echo "Unsupported container runtime"
exit 1`

for cases where cri isnt containerd. maybe make a helper function that has those 2 lines and call it in all the else statements?

Reply:
L145 already will fail, if runtime is not containerd. Having that check earlier prevents the need from repeating it right?

Reply:
should we just do this once at the very beginning of the build and stop bothering to check elsewhere?

Reply:
> L145 already will fail, if runtime is not containerd. Having that check earlier prevents the need from repeating it right?

I probably just missed that :D